tests/test_load_simul.py

```diff
@@ -0,0 +1,13 @@
+import pytest
+
+from snek5000 import load_simul
+
+
+def test_load_simul_error(sim_data):
+    with pytest.raises(ValueError):
```

Comment:
Couldn't we instead load the specified session? i.e. `load_simul(sim_data / "session_00")` equivalent to `load_simul(sim_data, session_id=0)`.

Reply:
It is possible, but I don't think I can support that in an elegant way, without providing some more checks and repeating  code. If the intention is to iterate over session directories, a simple function can achieve that.

```py
def get_sessions_ids(path_dir):
    path_dir = Path(path_dir)
    return (int(session.name.split("_")[-1]) for session in path_dir.glob("session_*"))
```

Reply:
I think one can detect this case with a quite simple check at the beginning of `load_simul`. Something like (maybe slightly more complicated):

```python
if session_id is None and path_dir.name.startswith("session_") and (path_dir.parent / "info_solver.xml").exists():
    session_id = int(path_dir.name[len("session_"):])
    path_dir = path_dir.parent
```

I guess users will tend to use `sim = load()` in the session directories and we should support that.

Reply:
I implemented this in 0f94728. Please take a look.

Reply:
Great! I think it's actually elegant! :slightly_smiling_face: